orienteer-bpm/src/main/java/org/orienteer/bpm/Initializer.java

```diff
@@ -18,6 +18,7 @@ public void init(Application application) {
 	@Override
 	public void destroy(Application application) {
 		OrienteerWebApplication app = (OrienteerWebApplication)application;
+		app.unregisterModule(BPMModule.class);
```

Comment:
We need to test there. Because it's assumed, that "register" is register in the system - disregard of shutdown, restart and etc.  And module has his own method onDestroy - which is called when application is shutting down.